Components/Button/Button.css

```diff
@@ -0,0 +1,73 @@
+.primary-btn{
+    padding: var(--std-btn-padding);
+    margin: 4rem;
+    border-radius: var(--rem-border-radius);
+    border-style: none;
+    background-color: var(--primary-font-color);
+    color: var(--white-font-color);
+    font-size: 2rem;
+    cursor: pointer;
+}
+
+.button-container{
+    background-color: var(--landing-hover-color);
+    padding: 3rem 5rem;
+    text-align: center;
+}
+
+.signup-btn{
+    padding: var(--std-btn-padding);
+    border-style: none;
+    border-radius: var(--rem-border-radius);
+    /* border: 2px solid var(--pink-border-color); */
+    font-size: 1.5rem;
+    color: var(--pink-border-color);
+    font-weight: bold;
+    border: 1px solid var(--inactive-color);
+    background-color: var(--white-font-color);
+    cursor: pointer;
+}
+
+.signup-btn:hover{
+    border: 2px solid var(--pink-border-color);
+}
+
+.link-btn{
```

Comment:
Underline don't look good for links. You can use text-decoration: none.

Reply:
i made that explicitly.